build.gradle

```diff
@@ -0,0 +1,20 @@
+plugins {
+    id 'java'
+}
+
+group 'com.alliander.equigy'
+version '0.1.0'
+
+sourceCompatibility = 1.8
```

Comment:
Voor OS vind ik het persoonlijk belangrijk om de compatability te vermelden in de readme of contributing?